### PR TITLE
Improve interface visibility in setting & library & pdfreader

### PR DESCRIPTION
**Pull request by Jack251970**

### CODE REVIEW

**Comment by HuntFeng** [comment]
At the first glance, They seem pretty good! 

I will check these changes in detail after the weekend (sorry, my wife doesn't like me doing work in weekend), should be able to merge this in the next few days. 

Happy weekend, and thank you for supporting research helper!

**Comment by Jack251970** [comment]
Thank you! These commits only cover small fixes and little interface improvements from my view.
I really like this project and I want to help its development, but it seems that I needs to learn more about Vue & electron.😢
Hope I can help more in the future! 🙂

File: quasar.config.js [hunk 1]
```diff
@@ -207,15 +207,19 @@ module.exports = configure(function (/* ctx */) {
         },
         linux: {
           target: "AppImage",
+          category: "Utility",
         },
         win: {
           target: "nsis",
         },
         nsis: {
+          oneClick: false,
```
Comment (HuntFeng) [inline quasar.config.js:216]: Unfortunately, according to electron-builder's [documentation](https://www.electron.build/configuration/nsis), `deleteAppDataOnUninstall` is for one-click installer only.

> `deleteAppDataOnUninstall` = false Boolean - one-click installer only. Whether to delete app data on uninstall.

I like the idea that user can choose their install path, it is a feature I want as well. To resolve this conflict, I need to do some research on the uninstall-script for all platforms. 

An easier way is to remove the `deleteAppDataOnUninstall` option, I will write some detail steps about how to remove user data on documentation page and let users to figure out themselves.

**Review by HuntFeng — COMMENTED** [review]
A very nice pull request, I like the changes, good job! Only some tiny fixes before the pull request can merge (see the comments).

Here is a nice documentation on the built-in classes for spacing in Quasar framework, with this you can save some of works. Of course, the spacing classes are very limited, if you like your styles, just keep them.

[Quasar's built-in class for spacing](https://quasar.dev/style/spacing#table-of-permutations)

Have a nice weekend!

**Comment by Jack251970** [comment]
Thank you for helping me with those basic problems.
I'm not familiar with the Quasar, and I'm not good at front-end technology, so sorry for bothering you. 😫
But from my view, there seem to be interface bugs in 'zoom in' and 'enter full screen' button. 🧐

**Comment by HuntFeng** [comment]
No worries, you did a good job! The merge will be approved once the changes are made. 

Could you raise an issue and describe the bug you mentioned?

**Comment by Jack251970** [comment]
> No worries, you did a good job! The merge will be approved once the changes are made.
> 
> Could you raise an issue and describe the bug you mentioned?

I have raise this issue.
Thanks for your work to check my commits! 👍

**Pull request closed without merging**